### PR TITLE
tools: add script to refresh gcp pro images

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
tools: add script to refresh gcp pro images

Add script that will look for updated gcp pro image ids and update our features/gcp-ids.yaml file with the latest version of the images

## Test Steps
Run the tools script and see if the image ids are updated
Run the GCP pro tests with the new image ids

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

**Review by renanrodrigo — APPROVED** [review]
This works, no CI involved so failures unrelated.
We even got a newer Bionic image than this here, if you want to update it before merging

**Comment by lucasmoura** [comment]
Awesome, will do

File: tools/refresh-gcp-pro-ids.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    for series, images in series_bucket.items():
+        latest_images[series] = "{}{}".format(
+            "projects/ubuntu-os-pro-cloud/global/images/", sorted(images)[-1]
+        )
+
+    with open("features/gcp-ids.yaml", "w") as stream:
+        stream.write(yaml.dump(latest_images, default_flow_style=False))
+
+    print(FOOTER_MSG)
```
[the remark this answers is not in the document]
Comment (lucasmoura) [reply tools/refresh-gcp-pro-ids.py:61]: Good point

**Comment by lucasmoura** [comment]
@orndorffgrant code updated

**Review by orndorffgrant — APPROVED** [review]
LGTM Thanks!

**Pull request merged**